src/templates/plugin.ejs

```diff
@@ -32,10 +26,10 @@ export default ({ app }, inject) => {
         }
     });
 
-    inject('getExtendedComponents', type => extendComponents[type] || {});
-    inject('extendMethods', async (type, payload) => {
-        if (extendMethods[type]) {
-            const methods = await Promise.all(extendMethods[type].map(method => method(payload)));
+    inject('getExtendSlot', key => extendComponents[key] || {});
```

Comment:
what about keeping same naming convention for extendComponents? extendSlots?

Reply:
`extendComponents` is a property that accepts keys that specify the names of the components that we want to extend, `extendComponents` will fit better.